protocol-contracts/staking/contracts/ProtocolStaking.sol

```diff
@@ -117,25 +115,27 @@ contract ProtocolStaking is AccessControlDefaultAdminRulesUpgradeable, ERC20Vote
     }
 
     /**
-     * @dev Unstake `amount` tokens from `msg.sender`'s staked balance to `recipient`.
-     * @param recipient The recipient where unstaked tokens should be sent.
      * @param amount The amount of tokens to unstake.
      * @return releaseTime The timestamp when the unstaked tokens can be released.
      *
+     * @dev Unstake `amount` tokens from `msg.sender`'s staked balance to `msg.sender`.
+     *
```

Comment:
Nit, for consistency:
```suggestion
     * @dev Unstake `amount` tokens from `msg.sender`'s staked balance to `msg.sender`.
     * @param amount The amount of tokens to unstake.
     * @return releaseTime The timestamp when the unstaked tokens can be released.
     *
```
(Or replace `@dev` on top with `@notice` and add other real `@dev` note)